swift-sdk/ITBConsts.swift

```diff
@@ -42,4 +32,7 @@ import Foundation
         public static let inputPlaceholder = ITBL_BUTTON_INPUT_PLACEHOLDER
         public static let action = ITBL_BUTTON_ACTION
     }
+
+    private static let apiHostname = "https://api.iterable.com"
```

Comment:
Any reason why this is separated from the endpoints?
Are you thinking of reusing these constants?

Reply:
I find it easier to switch to 'localhost:9000' etc when testing. I just have to change the hostname.